File: apollo/src/main/java/io/muun/apollo/domain/errors/InvalidPhoneNumberError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class InvalidPhoneNumberError extends UserFacingError {


    public InvalidPhoneNumberError() {
        super(UserFacingErrorMessages.INSTANCE.invalidPhoneNumber());
    }

    public InvalidPhoneNumberError(Throwable cause) {
        super(UserFacingErrorMessages.INSTANCE.invalidPhoneNumber(), cause);
    }
}

File: apollo/src/main/java/io/muun/apollo/domain/errors/InvalidPhoneNumberError.kt
package io.muun.apollo.domain.errors


import io.muun.apollo.data.external.UserFacingErrorMessages

class InvalidPhoneNumberError: UserFacingError {

    constructor():
        super(UserFacingErrorMessages.INSTANCE.invalidPhoneNumber())

    constructor(cause: Throwable):
        super(UserFacingErrorMessages.INSTANCE.invalidPhoneNumber(), cause)
}
